config/base.yml

```
@@ -203,8 +202,8 @@ script:
 
 after_success:
   # codecov
-  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then ./codecov-linux -f coverage.xml; fi
-  
+  - if [[ $PHPUNIT_COVERAGE_TEST ]] && [[ $CODECOV_VERIFIED ]]; then ./codecov -f coverage.xml; fi
+  - if [[ $PHPUNIT_COVERAGE_TEST ]] && ! [[ $CODECOV_VERIFIED ]]; then echo "codecov verification was INVALID!"; fi
```

That's not going to fail the build thought. You need to return an error code.

---

```suggestion
  - if [[ $PHPUNIT_COVERAGE_TEST ]] && ! [[ $CODECOV_VERIFIED ]]; then echo "codecov verification was INVALID!" && false; fi
```

---

Updated